libs/gl-client/src/lnurl/pay/mod.rs

```diff
@@ -1,25 +1,25 @@
-pub mod models;
-
+use super::models;
+use super::utils::parse_lnurl;
+use crate::lnurl::models::{
+    LnUrlHttpClient, MockLnUrlHttpClient, PayRequestCallbackResponse, PayRequestResponse,
+};
 use anyhow::{anyhow, ensure, Result};
-use bech32::FromBase32;
+use futures::future;
 use lightning_invoice::{Invoice, InvoiceDescription};
 use log::debug;
-use models::{
-    LnUrlHttpClient, MockLnUrlHttpClient, PayRequestCallbackResponse, PayRequestResponse,
-};
 use reqwest::Url;
 use sha256;
-use std::str::FromStr;
+use std::str::FromStr; // Import the future module
```

Comment:
The comment seems to have slipped between the `use` statements?

Reply:
Removed